room_with_a_view/settings.yaml.default

```diff
@@ -0,0 +1,10 @@
+connections:
```

Comment:
.default -> .example?

Reply:
`default` is special and intentional, if you don't pass a command-line argument whatever is in `default` will be used (similar to Django's handling of DB config).